src/LaunchDarkly/Integrations/DynamoDb.php

```diff
@@ -22,12 +29,14 @@ class DynamoDb
      *   - `dynamodb_options`: can include any settings supported by the AWS SDK client
      *   - `dynamodb_prefix`: a string to be prepended to all database keys; corresponds to the prefix
      * setting in ld-relay
+     *   - `dynamodb_client`: an already-configured DynamoDb client instance if you wish to reuse one; if
+     * specified, this will cause all other options except `dynamodb_prefix` and `dynamodb_table` to be ignored
      *   - `apc_expiration`: expiration time in seconds for local caching, if `APCu` is installed
-     * @return mixed  an object to be stored in the `feature_requester` configuration property
+     * @return \Closure  an object to be stored in the `feature_requester` configuration property
      */
-    public static function featureRequester(array $options = array())
+    public static function featureRequester(array $options = [])
     {
-        return function ($baseUri, $sdkKey, $baseOptions) use ($options) {
+        return static function ($baseUri, $sdkKey, $baseOptions) use ($options) {
```

Comment:
I think this modification is technically a breaking change because it affects how the `bindTo` functionality works with the return value of this method.

Can we revert this piece of it?

Reply:
Right! haven't though about it, on it. I'm also pushing a client instance validation.